packages/wrap/src/transforms/TransformInputObjectFields.ts

```diff
@@ -165,7 +167,10 @@ export default class TransformInputObjectFields<TContext = Record<string, any>>
             // The casting is kind of legit here as we are in a visitor
             const parentType = typeInfo.getInputType() as Maybe<GraphQLInputObjectType>;
```

Comment:
We shouldn't do casting here if we have the following check right?

Reply:
`getInputType` returns a `GraphQLInputType` instead of `GraphQLInputObjectType`, which does not have the property `name`. So on the ternary operator if the check fails it falls back to the original logic of just `parentType.name`. So I am not sure how we could do this without casting it here

Reply:
Removed with the `getNamedType` change